R/update-news.R

```diff
@@ -45,6 +45,17 @@ remove_housekeeping <- function(message) {
 }
 
 extract_newsworthy_items <- function(message) {
+  if (is_merge_commit(message)) {
+    title <- harvest_pr_title(message)
```

Comment:
We want the PR number to be extracted automatically, we should also be able to do it offline:

```suggestion
    pr_data <- harvest_pr_data(message) # tibble or NULL
    title <- pr_data$title
    pr_num <- pr_data$pr_num
```